benchflow-dsl/src/test/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/service/ServiceExplorationSpaceTest.scala

```diff
@@ -0,0 +1,39 @@
+package cloud.benchflow.dsl.definition.configuration.goal.explorationspace.service
+
+import org.junit.{ Assert, Test }
+import org.scalatest.junit.JUnitSuite
+import net.jcazevedo.moultingyaml._
+import cloud.benchflow.dsl.definition.configuration.goal.explorationspace.service.ServiceExplorationSpaceYamlProtocol._
+
+import scala.util.Try
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-23
+ */
+class ServiceExplorationSpaceTest extends JUnitSuite {
+
+  private val serviceExplorationSpace: String =
+    """
+      |resources:
+      |  memory:
+      |    range: [1g,5g]
+      |environment:
+      |  AN_ENUM: [value1, value2, value3]
+    """.stripMargin
+
+  @Test def completeUserExplorationSpace(): Unit = {
+
+    val workloadExplorationSpace = serviceExplorationSpace.parseYaml.convertTo[Try[ServiceExplorationSpace]]
+
+    Assert.assertTrue(workloadExplorationSpace.isSuccess)
+
+    val workloadExplorationSpaceYaml = workloadExplorationSpace.get.toYaml
+
```

Comment:
You should also assert you obtain the wanted configurations for the exploration space.